### PR TITLE
Added annotation typing to mnasnet

### PR DESCRIPTION
**Pull request by frgfm**
Hi there!

As per #2025, annotation typing are welcome in torchvision. So, this PR focuses on `torchvision.models.mnasnet` this PR !

Any feedback is welcome!

### CODE REVIEW

File: torchvision/models/mnasnet.py [hunk 1]
```diff
@@ -1,8 +1,11 @@
 import warnings
 
 import torch
+from torch import Tensor
 import torch.nn as nn
 from .utils import load_state_dict_from_url
+from torch.jit.annotations import Dict, List
```
Comment (fmassa) [inline torchvision/models/mnasnet.py:7]: Given that we now use Python3, you can now import `Dict` and `List` from `typing`.

Comment (fmassa) [reply torchvision/models/mnasnet.py:7]: Also, this is not very important as a change, but it is a nice cleanup that could be done

Comment (frgfm) [reply torchvision/models/mnasnet.py]: Oh I thought it was for jittability reasons on those specific types. I'll update the other PRs too then!

**Review by fmassa — APPROVED** [review]
Thanks!

**Pull request merged**